Sources/ProjectSpec/TargetSource.swift

```diff
@@ -12,6 +12,7 @@ public struct TargetSource: Equatable {
     public var type: SourceType?
     public var optional: Bool
     public var buildPhase: BuildPhase?
+    public var copyFiles: CopyFiles?
```

Comment:
Instead of adding another field, I’m going to make `BuildPhase` have an associated value. It makes my follow up PR easier as well as cleaning up the paths that don’t care about copyFiles.